main/xf86-video-s3virge/APKBUILD

```diff
@@ -1,17 +1,17 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=xf86-video-s3virge
 pkgver=1.10.7
-pkgrel=3
+pkgrel=4
 pkgdesc="S3 ViRGE video driver"
-url="http://xorg.freedesktop.org/"
+url="https://xorg.freedesktop.org/"
 arch="all"
 license="MIT"
 subpackages="$pkgname-doc"
 depends=
 makedepends="xorg-server-dev libxi-dev util-macros xorgproto"
-source="http://www.x.org/releases/individual/driver/$pkgname-$pkgver.tar.bz2
+source="https://www.x.org/releases/individual/driver/$pkgname-$pkgver.tar.bz2
+	check-max[HV]Value.patch
```

Comment:
can you please rename the patch file to not include `[` or `]` chars? I believe its generally a bad idea to include those chars in a file name, just like it is a bad idea to include chars like `?` and `*` in filenames.

Reply:
Noted and fixed.